stdlib/influxdata/influxdb/influxdb.flux

```diff
@@ -287,6 +287,24 @@ builtin from : (
 // a,tag2=b hum=53.5,temp=98.6 1609459380000000000
 // ```
 //
+// ### Write to multiple InfluxDB buckets
+// The example below does the following:
+//
+// 1. Writes data to `bucket1` and returns the data as it is written.
+// 2. Ungroups the returned data.
```

Comment:
"Ungroups" with `group()`? That may warrant more explanation?

Reply:
Added a clearer explanation here.